### ISSUE TITLE
file "keys.txt" in 01 - Inspect data and add geometry

### ISSUE THREAD

**Issue by pnkrtm** [rmorel/Viking#2]
Hello! I'm trying to run your notebooks one after one, and I found some problems in the second notebook "01 - Inspect data and add geometry.ipynb". It can't find file "keys.txt". Should it be in the repo?
Thanks for the answer!

**Comment by rmorel** [comment]
Indeed. I forgot to add it to the repository. I'll fix it as soon is I can.

**Comment by rmorel** [comment]
I added the file keys.txt and cleaned some older code. Please use the notebooks in the "notebooks" folder.

If you can, check and give me a heads up if everything is working.

**Comment by pnkrtm** [comment]
Yes, I checked it and this problem is gone!:)

Unfortunately, I found another one in the last notebook "07 - Mute linear events and stack.ipynb". Cell, which id writing output muted picks to file "mute_picks.rsf" does it badly. For this reason, the next cell can't run correctly and finishes with the problem "sfspray: build/api/c/file.c: trouble reading: 0 of 480". I've solved this problem by reordering cells and putting this cell before the reading data from "receiver_deghost.rsf". After that everything is clear!
